### DISCUSSION TITLE
Why is Slider value a number array instead of just a number?

### DISCUSSION THREAD

**Discussion by trgsv** [huntabyte/shadcn-svelte#217] [Help]
This makes implementation weird and is very unusual.

**Comment by agnostic-coder** [comment]
I think the radix implementation is maintained to support range sliders as well
see https://www.radix-ui.com/primitives/docs/components/slider#track

**Comment by huntabyte** [comment]
It's for handling multiple thumbs. I do think it may be worth raising an issue with https://github.com/melt-ui, as one would think the value would be the difference between the two thumb values 🤔.

**Comment by SikandarJODD** [comment]
how to bind value with slider ..this sucks a lotttt please provide example using bind ..
